lib/u3d/commands_generator.rb

```diff
@@ -148,6 +148,15 @@ def run
         end
       end
 
+      command :dependencies do |c|
+        c.syntax = 'u3d dependencies [-p | --package_manager <package manager>]'
+        c.description = 'Installs Unity dependencies. [Linux only]'
+        c.option '-p', '--package_manager STRING', String, 'Specify which package manager to use (aptitude, rpm...)'
```

Comment:
option is unused right now, correct? If so remove it.

Reply:
Proposed implementation, currently in this PR:

```
def self.install(prefix: nil)
      unless prefix
        if `which dpkg` != ''
          prefix = 'apt-get -y install'
        elsif `which rpm` != ''
          prefix = 'yum -y install'
        else
          raise 'Cannot install dependencies on your Linux distribution'
        end
      end
      if UI.interactive?
        return unless UI.confirm "Install dependencies? (#{DEPENDENCIES.length} dependency(ies) to install)"
      end
      U3dCore::CommandExecutor.execute(command: "#{prefix} #{DEPENDENCIES.join(' ')}", admin: true)
    end
```